From b84775d015dd873d718b2f8447206e71129e54c6 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Sun, 27 Jan 2019 16:27:04 -0800
Subject: [PATCH 10/12] Settings: Move artwork and visualizer to "Lock screen
 preferences"

These aren't related to the security on the lock screen so move them
out of the security settings page and into the actual lock screen
settings page.

Change-Id: I2e3fc672ef5d4816620a2343796177131b4461bd
---
 res/xml/screen_lock_settings.xml         | 10 ----------
 res/xml/security_lockscreen_settings.xml | 10 ++++++++++
 2 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/res/xml/screen_lock_settings.xml b/res/xml/screen_lock_settings.xml
index d1895d9aa5..9c7607043f 100644
--- a/res/xml/screen_lock_settings.xml
+++ b/res/xml/screen_lock_settings.xml
@@ -63,14 +63,4 @@
         android:title="@string/unlock_scramble_pin_layout_title"
         android:summary="@string/unlock_scramble_pin_layout_summary" />
 
-    <lineageos.preference.LineageSecureSettingSwitchPreference
-        android:key="lockscreen_media_metadata"
-        android:title="@string/lockscreen_media_art_title"
-        android:defaultValue="true" />
-
-    <lineageos.preference.LineageSecureSettingSwitchPreference
-        android:key="lockscreen_visualizer"
-        android:title="@string/lockscreen_visualizer_title"
-        android:defaultValue="true" />
-
 </PreferenceScreen>
diff --git a/res/xml/security_lockscreen_settings.xml b/res/xml/security_lockscreen_settings.xml
index 7cad34a13c..0a589f36ac 100644
--- a/res/xml/security_lockscreen_settings.xml
+++ b/res/xml/security_lockscreen_settings.xml
@@ -34,6 +34,16 @@
         android:title="@string/owner_info_settings_title"
         android:summary="@string/owner_info_settings_summary" />
 
+    <lineageos.preference.LineageSecureSettingSwitchPreference
+        android:key="lockscreen_media_metadata"
+        android:title="@string/lockscreen_media_art_title"
+        android:defaultValue="true" />
+
+    <lineageos.preference.LineageSecureSettingSwitchPreference
+        android:key="lockscreen_visualizer"
+        android:title="@string/lockscreen_visualizer_title"
+        android:defaultValue="true" />
+
     <SwitchPreference
         android:key="security_setting_lockdown_enabled"
         android:title="@string/lockdown_settings_title"
-- 
2.17.1

